packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -58,6 +60,7 @@ export const SignInFactorOnePasswordCard = (props: SignInFactorOnePasswordProps)
   const clerk = useClerk();
 
   const goBack = () => {
+    card.setError(undefined);
```

Comment:
Maybe we ignore `undefined` ? Or maybe there's a function to clear the error? Could you check the implementation?

Reply:
It worked when I moved it to the proper back handlers.